src/rgw/motr/gc/gc.cc

```diff
@@ -136,6 +145,24 @@ void MotrGC::initialize() {
   // set random starting index for enqueue of delete requests
   enqueue_index = \
     ceph::util::generate_random_number(0, max_indices - 1);
+  // Init KV lock provider
+  std::unique_ptr<MotrLockProvider> kv_lock_provider =
+      std::make_unique<MotrKVLockProvider>();
+  if (rc == 0) {
```

Comment:
which rc you are referring here ?
L-128 - that will be always false , see the codacy error
L-134 - that scope is over

Reply:
Removed duplicate variable 'rc'. Declared one at function level scope